app/assets/styles/partials/objects/_page.scss

```diff
@@ -47,8 +48,10 @@
 
 .page__content {
   flex: 1 0 auto;
+  padding-bottom: 1px;
```

Comment:
Can we at least add a comment in the code with a link to this bug so we know why this seemingly arbitrary bit of padding his here?